src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -265,7 +262,7 @@ void manualRegistrationToolBox::stopManualRegistration()
 
 void manualRegistrationToolBox::synchroniseMovingFuseView()
 {
-    if(this->parentToolBox()) //no need in Reg. workspace
+    if(selectorToolBox()) //no need in Reg. workspace
```

Comment:
In fact, the only functions where we need to test `selectorToolBox()` are those which are run at the software close. It seems to be mostly the functions handling change of input in comboboxes.